components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -1431,12 +1432,40 @@ void RewardsServiceImpl::SetCatalogIssuers(const std::string& json) {
   bat_ledger_->SetCatalogIssuers(json);
 }
 
+std::vector<uint64_t> RewardsServiceImpl::GetEarningsRange() {
+  uint64_t difference;
+  auto base_time_now = base::Time::Now();
+  int one_day = (60 * 60 * 24);
+
+  base::Time::Exploded exploded;
+  base_time_now.LocalExplode(&exploded);
+
+  if (exploded.month > 5) {
+    difference = ((exploded.month - 5) * one_day);
+  } else {
+    difference = (exploded.month * one_day);
+  }
+
+  uint64_t now = std::time(nullptr);
```

Comment:
std::time should be cast to uint64_t, i.e. `uint64_t now = static_cast<uint64_t>(std::time(nullptr))`

Reply:
done